### PR TITLE
add etcd length test

### PR DESCRIPTION
**Pull request by OmerBenHayun**

### CODE REVIEW

File: etcd.go [hunk 1]
```diff
@@ -12,6 +12,8 @@ import (
 	"go.etcd.io/etcd/clientv3"
 )
 
+const validKeyValueBytes = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
```
Comment (roytman) [inline etcd.go:15]: where do you use it ?

File: tests/etcd_length_test/ip_list/ips_list.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+cluster
```
Comment (roytman) [inline tests/etcd_length_test/ip_list/ips_list.txt]: I'd remove this file from PD, it contains internal IPs

File: tests/etcd_length_test/script/etcd_length_test_script.bash [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+ext_ip=`curl ifconfig.me`
+echo "my external ip is ${ext_ip}"
+echo "my internal ip is ${int_ip}"
+etcd --name="${name}" \
+     --initial-advertise-peer-urls http://${int_ip}:2380 \
+     --listen-peer-urls http://${int_ip}:2380 \
+     --listen-client-urls http://${int_ip}:2379,http://127.0.0.1:2379 \
+     --advertise-client-urls http://${int_ip}:2379 \
+     --initial-cluster-token etcd-cluster-1 \
+     --initial-cluster perf-etcd-aot-1=http://10.243.64.8:2380,perf-etcd-aot-2=http://10.243.64.9:2380,perf-etcd-aot-3=http://10.243.64.10:2380 \
```
Comment (roytman) [inline tests/etcd_length_test/script/etcd_length_test_script.bash:13]: please mask internal IPs

File: etcd.go (hunk 1 above)
Comment (OmerBenHayun) [reply etcd.go:15]: We use it on `randString` method In order to generate random strings that consist only from the characters on `validKeyValueBytes `:
```go
func (ks *randSpace) randString() string {
	p := make([]byte, ks.nbytes)
    for i := range p {
        p[i] = validKeyValueBytes[rand.Intn(len(validKeyValueBytes))]
    }
	return string(p)
}
```

File: tests/etcd_length_test/ip_list/ips_list.txt (hunk 2 above)
Comment (OmerBenHayun) [reply tests/etcd_length_test/ip_list/ips_list.txt]: this file removed from the PR.

**Pull request merged**